src/buttons/addChannel.ts

```diff
@@ -1,12 +1,17 @@
-import { ButtonInteraction, MessageActionRow, MessageButton } from 'discord.js';
+import {
+    ButtonInteraction,
+    ActionRowBuilder,
+    ButtonBuilder,
+    ButtonStyle,
```

Comment:
Comma

Reply:
This is correct use of trailing comma trailing comma is used in case you have large lists for example this has trailing comma:
```
One,
Two,
Three,
```
This does not have trailing comma
```
One, Two, Three
```